.circleci/config.yml

```diff
@@ -55,17 +56,23 @@ jobs:
               set -e
             }
 
-            # Push a tag and a latest if tagged
-            if [ -n "${CIRCLE_TAG}" ]; then
+            if [ "${DOCKER_USER}" == "" ] || [ "${DOCKER_PASS}" == "" ]; then
+              echo "Skipping Login to Dockerhub, credentials not available."
+            else
               echo "${DOCKER_PASS}" | docker login -u="${DOCKER_USER}" --password-stdin
 
-              # Push tagged image
-              retry docker tag "local/socorro-minidump-stackwalk:latest" "mozilla/socorro-minidump-stackwalk:${CIRCLE_TAG}"
-              retry docker push "mozilla/socorro-minidump-stackwalk:${CIRCLE_TAG}"
+              export LOCAL_IMAGE="local/socorro-minidump-stackwalk:latest"
+              export DOCKERHUB_REPO="mozilla/socorro-minidump-stackwalk"
```

Comment:
These are the variables we use in other projects. This makes it easier to copy-and-paste. Plus it makes it easier to read.